refinery/core/tests.py

```diff
@@ -2003,13 +2004,35 @@ def test_dataset_metadata_edit(self):
         pass  # TODO
 
     def test_dataset_visualization_creation(self):
-        pass  # TODO
+        create_tool_with_necessary_models("VISUALIZATION")
+
+        events = Event.objects.all()
+        self.assertEqual(len(events), 2)
+        self.assertRegexpMatches(
+            str(events[0]),
+            self.pre_re + r'created' + self.post_re
+        )
+        self.assertRegexpMatches(
+            str(events[1]),
+            self.pre_re + r'launched visualization None on' + self.post_re
```

Comment:
The `None` here and below confuse me. It could be that you may not have the latest develop code, but I thought that `Tools` got a reasonable default for their `display_names`.

Reply:
Let me look into that... It might be that I'm not filling in the object correctly, and that `None` is the actual value, and not the `str` for some object.